src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -323,18 +325,20 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
          !$omp parallel
          !$omp do schedule(runtime) private(k)
```

Comment:
`thicknessSum` needs to be private here

Reply:
Thanks, that solved the threading problem.